experimental/PropertyDDS/services/property-query-service/package.json

```diff
@@ -55,7 +55,7 @@
 		"async": "^3.2.2",
 		"basicauth-middleware": "2.0.0",
 		"body-parser": "^1.18.3",
-		"chalk": "2.4.1",
+		"chalk": "^2.4.2",
```

Comment:
We have rules that ensure each workspace uses the same versions of any deps. We also use `^` deps generally unless there is a reason not to. So rather than make my new dep match this, I opted to go the other way. So long as nothing breaks, this better matches our standard practices.